wdl-core/src/concern/lint/group.rs

```diff
@@ -9,6 +9,9 @@ pub enum Group {
     /// Rules associated with having a complete document.
     Completeness,
 
+    /// Rules associated with the whitespace in a document.
```

Comment:
Same as #13, I would personally still view this as `Style` rather than creating a new category for spacing specifically.

Reply:
I see now that you were going off of proposed groups in [this discussion](https://github.com/stjude-rust-labs/wdl/discussions/11). Given that we've thought this through at a higher level, I'm okay with keeping this group now.

Reply:
Yes, `Spacing` was described [here](https://github.com/stjude-rust-labs/wdl/blob/main/lint-rules.md) and this rule is tagged as being part of this group.

Reply:
Echoing what I said [here](https://github.com/stjude-rust-labs/wdl/pull/13#discussion_r1571425459), I am fine with either `Style` or `Spacing`. My desire is to eventually have this rule tagged with **both**. Until that's enabled, this seems an unimportant distinction to me.